### ISSUE TITLE
grid_buysell_advanced.py

### ISSUE THREAD

**Issue by Holender26** [Holender26/Scalp-bot#8]
cat > grid_buysell_advanced.py <<'PY'
import json, time, os, csv, math
from collections import deque
from datetime import datetime, timezone
import argparse
import requests

BINANCE_PRICE_URL = "/api/v3/ticker/price"

# ===== ANSI COLORS =====
RST = "\033[0m"
BOLD = "\033[1m"
DIM = "\033[2m"
RED = "\033[31m"
GRN = "\033[32m"
YEL = "\033[33m"
BLU = "\033[34m"
MAG = "\033[35m"
CYN = "\033[36m"

def c(text, color): return f"{color}{text}{RST}"

def now_utc():
    return datetime.now(timezone.utc).strftime("%Y-%m-%d %H:%M:%S")

def load_config(path):
    with open(path, "r", encoding="utf-8") as f:
        return json.load(f)

def to_binance_symbol(symbol: str) -> str:
    return symbol.replace("/", "").upper().strip()

def get_price(symbol_binance: str, base_url: str) -> float:
    url = base_url.rstrip("/") + BINANCE_PRICE_URL
    r = requests.get(url, params={"symbol": symbol_binance}, timeout=10)
    r.raise_for_status()
    return float(r.json()["price"])

def ensure_csv(path: str, header):
    if os.path.exists(path):
        return
    with open(path, "w", newline="", encoding="utf-8") as f:
        csv.writer(f).writerow(header)

def append_csv(path: str, row):
    with open(path, "a", newline="", encoding="utf-8") as f:
        csv.writer(f).writerow(row)

def pick_active_strategy(strategies: dict) -> str:
    for name, cfg in strategies.items():
        if isinstance(cfg, dict) and cfg.get("enabled") is True:
            return name
    return "none"

def clamp(x, lo, hi):
    return max(lo, min(hi, x))

def pct(x):  # x=0.001 -> 0.10%
    return f"{x*100:.2f}%"

def main():
    ap = argparse.ArgumentParser()
    ap.add_argument("--config", required=True)
    ap.add_argument("--profile", choices=["safe", "aggressive"], default="safe")
    ap.add_argument("--mode", choices=["spot", "futures_sim"], default="spot")
    args = ap.parse_args()

    cfg = load_config(args.config)
    if cfg.get("mode") != "paper":
        print(c("Ten skrypt jest PAPER-only. Ustaw mode=paper w config.", RED))
        return

    ex = cfg.get("exchange", {})
    base_url = ex.get("base_url", "https://api.binance.com")

    market = cfg.get("market", {})
    symbol = market.get("symbol", "BTC/USDT")
    symbol_b = to_binance_symbol(symbol)

    risk = cfg.get("risk", {})
    max_daily_loss_pct = float(risk.get("max_daily_loss_pct", 5.0))
    cooldown_after_loss = int(risk.get("cooldown_seconds_after_loss", 120))

    exec_cfg = cfg.get("execution", {})
    slippage_bps = float(exec_cfg.get("slippage_bps", 0))
    fee_bps = float(exec_cfg.get("fee_bps", 0))
    cost_bps = slippage_bps + fee_bps

    strategies = cfg.get("strategies", {})
    active = pick_active
